website/docs/commands/clean.md

```diff
@@ -3,4 +3,4 @@ title: Clean
 sidebar_position: 16
 ---
 
-The `clean` command deletes all of the files that are generated by `scala-cli`. This includes the `.scala-build` directory where Scala CLI writes outputs and stores its caches.
+The `clean` command deletes all of the files generated by `scala-cli`. This includes the `.scala-build` directory where Scala CLI writes outputs and stores its caches.
```

Comment:
...what's curious, we also clean the `.bsp` directory contents (if `scala-cli` created a connection file there), but not the `.bsp` directory itself (even if it's left empty afterwards). Dunno if it should be mentioned. And now I wonder if we intended this to be done in this precise manner.

but I digress.
```suggestion
The `clean` command deletes all of the files generated by `scala-cli`. This includes the `.scala-build` directory where Scala CLI writes outputs and stores its caches, and `.bsp/scala-cli.json` file which contains details for the BSP connection (necessary for importing a Scala CLI build to your IDE of choice).
```

Reply:
As long as it happens, it should be mentioned in the docs